app/chewy/media_index.rb

```diff
@@ -89,7 +91,9 @@ def get_streamers(type, ids)
       # Titles and freeform text
       translatable_field :titles
       field :abbreviated_titles, type: 'string'
-      field :synopsis, type: 'string', analyzer: 'english'
+      field :description do
+        field :en, type: 'string', analyzer: 'english'
+      end
```

Comment:
I think this will break the search functionality in the API 🤔

Reply:
how come?